deltachat-ios/Chat/DraftModel.swift

```diff
@@ -22,7 +22,7 @@ public class DraftModel {
     }
     var viewType: Int32? {
         if let viewType = draftMsg?.type {
```

Comment:
Why not just `return draftMsg?.type`? And then: Why not drop `viewType`-property in general? Reading this PRs description it seems to me, that that was the intention?

Reply:
removing viewType from DcMsg was the intention, i stopped when it comes to streaming other code

but yes, i also thought about these lines, will have a second look

Reply:
applied your suggestion, thanks!